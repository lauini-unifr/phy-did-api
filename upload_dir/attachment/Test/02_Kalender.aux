\relax 
\providecommand\hyper@newdestlabel[2]{}
\newlabel{chap_Kalender}{{3.2}{1}{Kalendersysteme}{section*.15}{}}
\@writefile{lot}{\contentsline {table}{\numberline {1}{\ignorespaces Die wichtigsten physikalischen Gr\"o\ss en im Zusammenhang mit den Kalendersystemen.}}{1}{table.1}}
\newlabel{tab_Kal}{{1}{1}{Die wichtigsten physikalischen Gr\"o\ss en im Zusammenhang mit den Kalendersystemen}{table.1}{}}
\@writefile{toc}{\contentsline {section}{\numberline {1}Tage, Monate und Jahre}{1}{section.1}}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.1}Tage}{1}{subsection.1.1}}
\newlabel{SC@14}{{1.1}{2}{Tage}{subsection.1.1}{}}
\@writefile{lof}{\contentsline {figure}{\numberline {1}{\ignorespaces Siderischer Tag und Sonnentag. Da sich die Erde im Verlauf eines Tages um den Winkel $\alpha $ weiterbewegt hat (hier \"ubertrieben dargestellt), muss sie sich relativ zum Fixsternhimmel um diesen Winkel weiter drehen, damit ein bestimmter Punkt wieder in Richtung Sonne zeigt.}}{2}{figure.1}}
\newlabel{fig_SiderischerTag}{{1}{2}{\SC@CAPtext }{figure.1}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.2}Monate}{2}{subsection.1.2}}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.3}Jahre}{3}{subsection.1.3}}
\@writefile{toc}{\contentsline {section}{\numberline {2}Der Menton-Zyklus und die Struktur von Mondkalendern}{4}{section.2}}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.1}Der J\"udische Kalender}{4}{subsection.2.1}}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.2}Der Islamische Kalender}{5}{subsection.2.2}}
\@writefile{toc}{\contentsline {section}{\numberline {3}Die Wochentage}{5}{section.3}}
\@writefile{lot}{\contentsline {table}{\numberline {2}{\ignorespaces Die Wochentage in verschiedenen europ\"aischen Sprachen und die zugeh\"origen Planeten.}}{5}{table.2}}
\newlabel{tab_Wochentage}{{2}{5}{Die Wochentage in verschiedenen europ\"aischen Sprachen und die zugeh\"origen Planeten}{table.2}{}}
\@writefile{toc}{\contentsline {section}{\numberline {4}Die Kalenderreform von 1582}{6}{section.4}}
\newlabel{sec_Kalender1582}{{4}{6}{Die Kalenderreform von 1582}{section.4}{}}
\@writefile{toc}{\contentsline {section}{\numberline {5}Kuriosit\"aten der Kalenderreform}{7}{section.5}}
\@setckpt{02_Kalender}{
\setcounter{page}{8}
\setcounter{equation}{2}
\setcounter{enumi}{4}
\setcounter{enumii}{0}
\setcounter{enumiii}{0}
\setcounter{enumiv}{5}
\setcounter{footnote}{2}
\setcounter{mpfootnote}{0}
\setcounter{part}{0}
\setcounter{section}{5}
\setcounter{subsection}{0}
\setcounter{subsubsection}{0}
\setcounter{paragraph}{0}
\setcounter{subparagraph}{0}
\setcounter{figure}{1}
\setcounter{table}{2}
\setcounter{parentequation}{0}
\setcounter{r@tfl@t}{0}
\setcounter{SC@C}{14}
\setcounter{Item}{39}
\setcounter{Hfootnote}{9}
\setcounter{bookmark@seq@number}{86}
\setcounter{num}{3}
\setcounter{section@level}{1}
}
