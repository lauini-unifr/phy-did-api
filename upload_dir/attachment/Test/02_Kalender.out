\BOOKMARK [1][-]{section.1}{Tage, Monate und Jahre}{}% 1
\BOOKMARK [2][-]{subsection.1.1}{Tage}{section.1}% 2
\BOOKMARK [2][-]{subsection.1.2}{Monate}{section.1}% 3
\BOOKMARK [2][-]{subsection.1.3}{Jahre}{section.1}% 4
\BOOKMARK [1][-]{section.2}{Der Menton-Zyklus und die Struktur von Mondkalendern}{}% 5
\BOOKMARK [2][-]{subsection.2.1}{Der J\374dische Kalender}{section.2}% 6
\BOOKMARK [2][-]{subsection.2.2}{Der Islamische Kalender}{section.2}% 7
\BOOKMARK [1][-]{section.3}{Die Wochentage}{}% 8
\BOOKMARK [1][-]{section.4}{Die Kalenderreform von 1582}{}% 9
\BOOKMARK [1][-]{section.5}{Kuriosit\344ten der Kalenderreform}{}% 10
